MapboxCoreNavigation/NavigationLocationManager.swift

```diff
@@ -9,38 +9,22 @@ import UIKit
 #endif
 
 /**
- `NavigationLocationManager` is the base location manager which handles
- permissions and background modes.
+ `NavigationLocationManager` is the base location manager which handles permissions and background modes.
+ 
+ If it's necessary to change the default behavior in `NavigationLocationManager`, you can subclass this class.
```

Comment:
In fact, the developer doesn’t even need to subclass this class to modify the desired accuracy: all they have to do is set `navigationViewController.locationManager.desiredAccuracy`.